### DISCUSSION TITLE
ChatGPT API Free proxy usable so can use this for free?

### DISCUSSION THREAD

**Discussion by Hedda** [ITSpecialist111/ai_automation_suggester#18] [Ideas]
Could one use "ChatGPT API Free" or other proxy to allow you to use this with OpenAI's ChatGPT API for free?

https://github.com/ayaka14732/ChatGPTAPIFree

**Comment by ITSpecialist111** [comment]
It appears that the API free idea has been archived. In version 1.07, it is now possible to use your own custom or local models. Since this setup does not require quick processing (only scheduling), you can utilize tools like Ollama to achieve similar goals.

**Changes in V1.07:**

1. **Removal of Automatic Scheduling:**
   - Automatic scanning and suggestion generation have been removed due to complexities surrounding triggers and user preferences.
   - This change enhances stability by allowing users to implement custom automation triggers that better suit their individual needs.

2. **Expanded AI Provider Support:**
   - **Support for Multiple LLM Models:**
     - Google LLM Models: Now supports Google's AI models for generating automation suggestions.
     - Local AI Models: Local models, such as LocalAI and Ollama, are now supported for users who prefer privacy and do not rely on internet connectivity.
   
   - **Custom LLM Variants:**
     - Users can now select custom AI model variants from higher-tier models (e.g., OpenAI's o1-preview), providing greater flexibility and control over the AI processing.

3. **Enhanced Manual Trigger Service:**
   - **Integration Selection:** 
     - Users can now choose which AI provider integration to trigger when manually initiating AI analysis.
   - **Custom System Prompt:** 
     - Users can override the built-in system prompt for more tailored and personalized suggestions.

4. **German Translation Added:**
   - **Global Reach:** 
     - German language support has been introduced to make the integration accessible to a broader audience.
